.eslintrc.json

```diff
@@ -8,6 +8,27 @@
 		"curly": "off",
 		"indent": "off",
 		"no-confusing-arrow": "off",
-		"no-mixed-operators": "off"
+		"no-mixed-operators": "off",
+		"import/no-extraneous-dependencies": [
+			"error", {
+				"devDependencies": [
+					"test/**",
+					"**/Gruntfile{,.js}" // grunt config
```

Comment:
Since this is a JSON file, comments can not be used.

Reply:
ah...yes, totally forgot it's not rc file =(

Reply:
ESLint allows comments in JSON.